custom_components/watergate/sensor.py

```diff
@@ -90,8 +90,8 @@ async def async_setup_entry(
             coordinator,
             WATER_METER_VOLUME_SENSOR_NAME,
             WATER_METER_VOLUME_ENTITY_NAME,
-            UnitOfVolume.MILLILITERS,
-            SensorDeviceClass.VOLUME,
+            UnitOfVolume.LITERS,
```

Comment:
@nivek1612 I think you cannot change that. We are providing the value in mililiter so you need to apply a conversion, right? And I cannot see it anywhere.

Reply:
That is interesting! We will figure it tomorrow.

Reply:
cool

Reply:
Ok I'll recheck post the firmware update. Let me know when it is pushed out

Reply:
@adam-the-hero just to check the value is a cumulative value. i.e. the total water volume since the integration started recording

Reply:
Yes it is total water volume registered by device. So basically it normal water meter.

Reply:
Will you change that? Or you want to do it after testing the new version of firmware?

Reply:
Let me test the new version. I changed it back in the old version and it didn’t work as expected